.kokoro/builddocs.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+cd ..
+
+# Build the libraries w/o running unit tests.
```

Comment:
Nit: "without" would be clearer than "w/o", and it's not like we're short of space.

Reply:
Done